README.md

```diff
@@ -0,0 +1,122 @@
+# Rainbow Button
+
```

Comment:
I think we should have a short description here:

Rainbow Button is a react component that renders an opinionated button built on top of [WalletConnect](https://walletconnect.org) v1 which allows you to connect to [Rainbow](https://rainbow.me) on mobile (via Mobile Deeplinking) and desktop (via QR codes)  and maybe add a recordit gif showing a quick demo